packages/bff/src/userApi.ts

```diff
@@ -0,0 +1,76 @@
+import { FastifyPluginAsync, FastifyReply, FastifyRequest } from 'fastify';
+import fp from 'fastify-plugin';
+import { ProfileRepository } from './db';
+import { Profile } from './entities/Profile';
+import { validateRequest } from './oidc';
+
+interface PluginOptions {
```

Comment:
Remove this.

Reply:
yup, brukte `fastiy`-type for dette (anbefalt i doc), så fjernet også i oidc